.github/workflows/lib-ms.yml

```diff
@@ -8,8 +8,6 @@ on:
     paths:
       - 'servers/lib/**'
   workflow_dispatch:
-    paths:
```

Comment:
Is there an advantage to remove this?

Reply:
It's not supposed to be there (It was marked as an error by my github actions linter)